pkg/monitoring/migration/prometheus.go

```diff
@@ -0,0 +1,150 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package migration
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	k8sv1 "k8s.io/api/core/v1"
+
+	virtv1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/log"
+)
+
+type gaugeAction bool
+
+const increase gaugeAction = true
+const decrease gaugeAction = false
+
+var (
+	migrationsLabels = []string{"vmi", "source", "target"}
+
+	CurrentPendingMigrations = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "kubevirt_migrations_pending",
```

Comment:
@machadovilaca Based on the kubevirt_migrate_vmi_data_remaining_bytes, kubevirt_migrate_vmi_data_processed_bytes and kubevirt_migrate_vmi_dirty_memory_rate_bytes metrics, I believe we should name this metric as kubevirt_migrate_vmi_pending_count

I would also consider changing the name of all migration metrics to start with
kubevirt_vmi_migrate_ prefix.

Reply:
done